<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCoursesDialog</class>
 <widget class="QDialog" name="SelectCoursesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>自由选课</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/choose.png</normaloff>:/icons/choose.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #CCCCFF;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="8,10,10,0">
   <item row="1" column="2" colspan="2">
    <widget class="QTextEdit" name="selectedCourses">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="showcourses_btn">
     <property name="text">
      <string>查看可选课程</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 0, 255);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="add_btn">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请选择当前感兴趣的课程：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="confirm_btn">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QListWidget" name="havedlist">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 170, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>已排课程</string>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
